### DISCUSSION TITLE
@Mike777Khoo 反馈 已能登录 谢谢

### DISCUSSION THREAD

**Discussion by xwyqsl** [getlantern/lantern-forum-cn#886] [Q&A]

**Comment by ct19820228** [comment]
今天付款的，下载的是这个测试版。
![微信图片_20240923191730](https://github.com/user-attachments/assets/1a8d2bdf-3d09-419f-8bcb-9e7dd2988e92)
一直没有发激活码。
也可以发我这个邮箱ct820228@gmail.com
